hypha/templates/base-apply.html

```diff
@@ -31,9 +31,7 @@
         <script src="{% static 'js/js.cookie.min.js' %}"></script>
         <script src="{% static 'js/main-top.js' %}"></script>
         <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
```

Comment:
Can we include alpinejs as a local file instead of using a CDN?

Reply:
I had tried to include it locally. It doesn't work, as it downloads a few other files and has something to do with how the library is loaded and initialized. I've to research it; we can create a separate ticket to track it. 

PS: This PR doesn't add alpine, it was added before so it's better we solve this in another PR.

Reply:
![Screenshot 2023-02-17 at 13 17 13@2x](https://user-images.githubusercontent.com/236356/219662923-999b533d-18c2-4643-adaf-e133d73c36c8.jpg)

Reply:
I tried to use the content on the js file locally, but it didn't work. By any chance, do you know if we can do module-based imports for javascript? https://alpinejs.dev/essentials/installation#as-a-module

Reply:
Instead, I've frozen the version of alpine to 3.11.x instead of 3.x. (in https://github.com/HyphaApp/hypha/pull/3227/commits/ffe855ad5a9c4ada3fe692e1f038a40643df64c5)

Reply:
Did some testing and we have an outdated "browserslist" that includes dead browsers like IE making babel not use modern presets. This break local alpinejs versions.

Updating to this (babel default I believe) fixes the issue.

~~~~
    "browserslist": [
        "> 0.5%",
        "last 2 versions",
        "Firefox ESR",
        "not dead"
    ],
~~~~

We should update to this in any case.

Reply:
Good catch! The updated browserslist lets alpine work! I've updated the codebase with the local version of alpine and the new browserlist.

Reply:
Good catch! The updated browserslist lets alpine work! I've updated the codebase with the local version of alpine and the new browserlist.